nvflare/private/fed/utils/app_deployer.py

```diff
@@ -25,49 +26,47 @@
 from .app_authz import AppAuthzService
 
 
-class AppDeployer(object):
-    def __init__(self, workspace: Workspace, job_id: str, job_meta: dict, app_name: str, app_data):
-        self.app_name = app_name
-        self.workspace = workspace
-        self.app_data = app_data
-        self.job_id = job_id
-        self.job_meta = job_meta
+class AppDeployer(AppDeployerSpec):
+    def __init__(self):
+        AppDeployerSpec.__init__(self)
```

Comment:
Since its base class is pure abstract, we don't need to call its init method hence we don't need to specify this init method here?